### PR TITLE
Actions to update the leaderboard

### PR DESCRIPTION
**Pull request by acolombo1**
- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/submit.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    if (nameInput.value !== '' && scoreInput.value !== '') {
+      const gameid = localStorage.getItem('gameid');
+      const uri = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameid}/scores/`;
+      try {
+        const response = await fetch(uri, {
+          method: 'POST',
+          body: JSON.stringify({ user: nameInput.value, score: scoreInput.value }),
+          headers: { 'Content-type': 'application/json; charset=UTF-8' },
+        });
```
Comment (Cathella) [inline src/modules/submit.js:18]: ✅ The submit button should hit the POST API endpoint and post user entries 👍🏻

File: src/modules/refresh.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    message('Please wait for the game ID to be loaded.');
+  } else {
+    // call api to get names/scores list
+    const gameid = localStorage.getItem('gameid');
+    const uri = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameid}/scores/`;
+    try {
+      const response = await fetch(uri, { method: 'GET' });
+      if (response.ok) {
+        const jsonresponse = await response.json();
+        // parse the data into the list
```
Comment (Cathella) [inline src/modules/refresh.js:14]: ✅ When the Refresh button is click, it should hit the GET API endpoint causing the scores list to change 👍🏻

File: src/index.js [hunk 3]
```diff
@@ -1 +1,15 @@
-import './style.css';
\ No newline at end of file
+/* eslint-disable linebreak-style */
```
Comment (Cathella) [inline src/index.js]: - [x] Kindly enable the linters and instead work on fixing the linter errors 🙏🏻. It helps with keeping consistent code standards

File: src/modules/createGame.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable linebreak-style */
```
Comment (Cathella) [inline src/modules/createGame.js]: - [x] Kindly enable the linters and instead work on fixing the linter errors 🙏🏻. It helps with keeping consistent code standards

File: src/modules/submit.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable linebreak-style */
```
Comment (Cathella) [inline src/modules/submit.js]: - [x] Kindly enable the linters and instead work on fixing the linter errors 🙏🏻. It helps with keeping consistent code standards

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @acolombo1 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct GitHub Flow.
✅ Professional README Structure.
✅ Descriptive pull request.
✅ App runs as expected. You can submit user data and it reflects on the scoreboard

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 3 above)
Comment (acolombo1) [reply src/index.js]: It's just the linebreak style that I don't want to show up all over in my VS Code. Then Git reverts them back to CRLF anyway. 
I removed the lines.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @acolombo1 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**